packages/@dcl/inspector/src/hooks/sdk/useComponentInput.ts

```diff
@@ -20,14 +20,16 @@ export const useComponentInput = <ComponentValueType extends object, InputType e
   component: Component<ComponentValueType>,
   fromComponentValueToInput: (componentValue: ComponentValueType) => InputType,
   fromInputToComponentValue: (input: InputType) => ComponentValueType,
-  isValidInput: (input: InputType) => boolean = () => true
+  validateInput: (input: InputType) => boolean = () => true,
+  deps: unknown[] = []
 ) => {
   const [componentValue, setComponentValue, isEqual] = useComponentValue<ComponentValueType>(entity, component)
   const [input, setInput] = useState<InputType | null>(
     componentValue === null ? null : fromComponentValueToInput(componentValue)
   )
   const [isFocused, setIsFocused] = useState(false)
   const [skipSync, setSkipSync] = useState(false)
+  const [monitor, setMonitor] = useState({ isValid: false })
```

Comment:
do we need to wrap this property in this object? The to use it we need to double unwrap it:

```ts
const { getInputProps, monitor: { isValid } } = useComponentInput()
```

When it could simply be another value at the same level:

```ts
const { getInputProps, isValid } = useComponentInput()
```

right?